testing/Dockerfile.connectsdk.test

```
@@ -0,0 +1,24 @@
+FROM node:19.6.1-slim@sha256:a1ba21bf0c92931d02a8416f0a54daad66cb36a85d2b73af9d73b044f5f57cfc
+
+RUN apt-get update && apt-get -y install \
+  git python3 make curl netcat
```

```suggestion
RUN apt-get update && apt-get -y install --no-install-recommends \
  git python3 make curl netcat && \
  apt-get clean && \
  rm -rf /var/lib/apt/lists/*
```

Put those images on a diet since you're not using a multi-stage build! Feel free to confirm the before and after with this trick:

```bash
docker image ls --format "{{ .Size }}" $IMAGE_NAME
```